app.rb

```diff
@@ -0,0 +1,19 @@
+
+
+require 'sinatra/base'
+require 'sinatra/flash'
+
+class ChitterChallenge < Sinatra::Base #basic sinatra application
+
+  register Sinatra::Flash #extension that lets you store information between requests.
+  enable :sessions #?
+
+
+  get '/' do # define a route
+    @user = User.find(session[:user_id])
+    @message = Peep.read
+  erb :index
+
```

Comment:
This view will not work at the moment as there is no User or Peep class, and no view setup for index